.github/workflows/preview-up.yml

```
@@ -68,3 +72,17 @@ jobs:
           path: logs/
           retention-days: 7
           if-no-files-found: warn
+
+      - name: Get preview environment URL
+        id: get-url
+        run: |
+          touch .env
+          PREVIEW_URL=$(npx preevy urls --json --wait --profile "$PREEVY_PROFILE_URL" | jq -r '.[0].url')
+          echo "✅ Using preview URL: $PREVIEW_URL"
+          echo "preview_url=$PREVIEW_URL" >> "$GITHUB_OUTPUT"
+
+  e2e-tests:
+    needs: deploy
+    uses: trade-tariff/trade-tariff-tools/.github/workflows/e2e-tests.yml@main
+    with:
+      test-environment: preview
```

So you can set the environment variables:
```suggestion
      test-environment: development
   env:
      BASE_URL: ${{ needs.deploy.outputs.preview_url }}
   secrets:
      basic-password: ${{ secrets.BASIC_PASSWORD }}
```